artikel/2015-07-31-Hitzewelle-Sommerspaß-oder-Klimawandel.md

```diff
@@ -0,0 +1,34 @@
+#Hitzewelle: Sommerspaß oder Klimawandel?
+
+http://www.dw.com/de/hitzewelle-sommerspa%C3%9F-oder-klimawandel/a-18616694
+
+der Artikel "Hitzewelle: Sommerspaß oder Klimawandel?" spricht ein
+interessantes und wichtiges Thema an, weil wir diesen Sommer so viel 
+Hitzwelle in Deutschland gehabt haben.
+
+Der Artikel geht um Klimawandel und die Möglichkeit, ob es existiert. In 
+2015 gab es rekorde Temperaturen und häufiger Hitzwelle in Deutschland. 
+Im Juli 2015 gab es einen neuen Hitzrekord in Bayern mit 40,3 Grad Celsius 
```

Comment:
Very good sentence and _almost_ perfect. Last part sounds a little strange with `gemessen`. I'd rather just omit `gemessen` and change the sentence to imparative style. Alternatively split it up into two short sentences.

First version:

    Im Juli 2015 gab es einen neuen Hitzerekord in Bayern mit 40,3 Grad Celsius!

Alternative version:

    Im Juli 2015 gab es einen neuen Hitzerekord in Bayern. Es wurden 40,3 Grad Celsius gemessen!